[DOCS] Install prototype (#3855)
percona/pmm: documentation/test.md

# Set up PMM Client

> To make Percona Monitoring and Management (PMM) work properly a **PMM Client** needs to be set up in your database environment. This client acts as the messenger, relaying back the key performance insights from your database to the PMM Server.

Reviewer: ```suggestion
To make the most of Percona Monitoring and Management (PMM), we suggest to set up a **PMM Client** in your database environment. This client acts as an agent, relaying back the key performance insights from your database to the PMM Server.
```

Depending on your setup, PMM Client may already be set up requiring only authentication or you may need to install it entirely. To understand what needs to be done in your system, select the database technology you're using first.

## Select your database technology and host

Select the database technology you're using so we can direct you to the best set up of your PMM Client and start collecting data signals back to PMM Server.

``` mermaid
graph RL
PS(PMM Server)
subgraph s2[Now: Step 2]
    DB[(Database)] -- Data collection --> PC(PMM Client)
end
PC -- Transmission --> PS
```

=== ":material-dolphin: MySQL"

    For MySQL check the type of host that you have for your database and follow the instructions required to set up PMM Client.

>     | <small>*Host*</small> | <small>*Recommended set up*</small> | <small>*Other advanced options*</small> |

Reviewer: IMO those captions look far too small and diffucult to read. I'd leave their regular size.
